cli/pcluster/cfnconfig.py

```diff
@@ -545,6 +548,26 @@ def __init_efa_parameters(self):
         except configparser.NoOptionError:
             pass
 
+    def __init_hyperthreading_parameter(self):
+        try:
+            __temp__ = self.__config.get(self.__cluster_section, "disable_hyperthreading")
+            if __temp__ not in  {"true", "false"}:
+                self.__fail("valid values for disable_hyperthreading = true/false")
+            # TODO: validate if instance type supports CPU Options
```

Comment:
this can be done in cfnconfig.py under the capacity check ec2 run-instance call.

Reply:
Fixed